MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+---
+layout: post
+title: Swiping in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Swiping support in Syncfusion .NET MAUI ListView (SfListView) control, its elements and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Swiping in .NET MAUI ListView (SfListView)
+
+## Overview
+
+The `SfListView` allows swiping items to do custom actions such as deleting the data, adding the data, editing the data, etc. To enable swiping, set the `SfListView.AllowSwiping` property to `true`. Swipe views are displayed when swiping from left to right or right to left (for `Horizontal` orientation, top to bottom or bottom to top) on the item. 
```

Comment:
The `SfListView` allows swiping items to achieve custom actions such as deleting the data, adding the data, editing the data, etc. To enable swiping, set the `SfListView.AllowSwiping` property to `true`. Swipe views are displayed when swiping from left to right or right to left (for `Horizontal` orientation, top to bottom, or bottom to top) on the item.

Reply:
resolved